ide/textmate.lexer/manifest.mf

```diff
@@ -0,0 +1,7 @@
+Manifest-Version: 1.0
+AutoUpdate-Show-In-Client: true
+OpenIDE-Module: org.netbeans.modules.textmate.lexer/0
```

Comment:
Why `/0`? Are you exposing any API in `org.netbeans.api` namespace? I don't see any.

Reply:
Devel API does not need to be in the org.netbeans.api package, or does it? Is it documented somewhere? I've found:
http://wiki.netbeans.org/API_Design#Devel
and it seems to allow Under Development APIs regardless the namespace.

(There are pages that say something like "if you want an API in the official namespace and you want to be able to change it, then the API must be "Under Development", but from that we can draw the conclusion in the opposite direction, of course.)